src/fides/data/sample_project/docker-compose.yml

```diff
@@ -31,6 +31,17 @@ services:
       - ${FIDES_DEPLOY_ENV_FILE:-sample.env}
     environment:
       FIDES__CONFIG_PATH: "/fides/src/fides/data/sample_project/fides.toml"
+      # Configure sample connection secrets for Postgres & Mongo (see services below)
+      FIDES_DEPLOY__CONNECTORS__POSTGRES__HOST: postgres-test
+      FIDES_DEPLOY__CONNECTORS__POSTGRES__PORT: 5432
+      FIDES_DEPLOY__CONNECTORS__POSTGRES__DBNAME: postgres_example
+      FIDES_DEPLOY__CONNECTORS__POSTGRES__USERNAME: postgres
+      FIDES_DEPLOY__CONNECTORS__POSTGRES__PASSWORD: postgres
+      FIDES_DEPLOY__CONNECTORS__MONGO_HOST: mongodb-test
+      FIDES_DEPLOY__CONNECTORS__MONGO_PORT: 27017
+      FIDES_DEPLOY__CONNECTORS__MONGO_DEFAULTAUTHDB: mongo_test
+      FIDES_DEPLOY__CONNECTORS__MONGO_USERNAME: mongo_user
+      FIDES_DEPLOY__CONNECTORS__MONGO_PASSWORD: mongo_pass
```

Comment:
These were previously defined in `scripts/setup/constants.py`, which was miles away from the sample project configuration. This should make it really clear how the secrets for these test databases are kept in sync with the service definitions themselves.